QualityControl/public/common/header.js

```diff
@@ -32,11 +31,10 @@ import { runModeHeader } from './runModeHeader.js';
  * @returns {vnode} - header element
  */
 export default (model) => h('.flex-col', [
-  h('.flex-row.p2.items-center', [
-    commonHeader(model),
-    headerSpecific(model),
+  h('.flex-row.p2.items-center', { id: 'qcg-header' }, [
+    h('.div.w-33', commonHeader(model)),
```

Comment:
There is no `.div` class. 
The convention is that we do not specify `div` by default as it is already assumed by default